src/message.ts

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Icon, IconDimension, IconName } from "@components/Icon";
+import { message as antdMessage } from "antd";
+
+interface MessageProps {
+	content: string;
+	duration?: number;
+}
+
+const defaultClassName = "";
+const defaultOnClose = () => antdMessage.destroy();
```

Comment:
Normally we shouldn't, no. But when the message is not destroyed, some elements remain in the DOM. With how the design is specified (the dark transparent background) this would cause the background to remain visible.
Destroying the message removes all DOM elements and prevents this from happening.

Reply:
alright. can you add that as a comment?

Reply:
Done!